language/de.json

```diff
@@ -0,0 +1,372 @@
+	"MRE Brave Art": "MRE Mutige Kunst",
+	"MRE Heroic Fantasy": "MRE Heroische Fantasie",
+	"MRE Dark Cyberpunk": "MRE Dunkler Cyberpunk",
+	"MRE Lyrical Geometry": "MRE Lyrische Geometrie",
+	"MRE Sumi E Symbolic": "MRE Sumi E Symbolisch",
+	"MRE Sumi E Detailed": "MRE Sumi E Detailliert",
+	"MRE Manga": "MRE Manga",
+	"MRE Anime": "MRE Anime",
+	"MRE Comic": "MRE Comic",
+	"Ads Advertising": "Werbung",
+	"Ads Automotive": "Automobilwerbung",
+	"Ads Corporate": "Unternehmenswerbung",
+	"Ads Fashion Editorial": "Modeeditorial",
+	"Ads Food Photography": "Lebensmittelfotografie",
+	"Ads Gourmet Food Photography": "Gourmet-Lebensmittelfotografie",
+	"Ads Luxury": "Luxuswerbung",
+	"Ads Real Estate": "Immobilienwerbung",
+	"Ads Retail": "Einzelhandelswerbung",
+	"Artstyle Abstract": "Kunststil Abstrakt",
+	"Artstyle Abstract Expressionism": "Kunststil Abstrakter Expressionismus",
+	"Artstyle Art Deco": "Kunststil Art Déco",
+	"Artstyle Art Nouveau": "Kunststil Jugendstil",
+	"Artstyle Constructivist": "Kunststil Konstruktivismus",
+	"Artstyle Cubist": "Kunststil Kubismus",
+	"Artstyle Expressionist": "Kunststil Expressionismus",
+	"Artstyle Graffiti": "Kunststil Graffiti",
+	"Artstyle Hyperrealism": "Kunststil Hyperrealismus",
+	"Artstyle Impressionist": "Kunststil Impressionismus",
+	"Artstyle Pointillism": "Kunststil Pointillismus",
+	"Artstyle Pop Art": "Kunststil Pop-Art",
+	"Artstyle Psychedelic": "Kunststil Psychedelisch",
+	"Artstyle Renaissance": "Kunststil Renaissance",
+	"Artstyle Steampunk": "Kunststil Steampunk",
+	"Artstyle Surrealist": "Kunststil Surrealismus",
+	"Artstyle Typography": "Kunststil Typografie",
+	"Artstyle Watercolor": "Kunststil Aquarell",
+	"Futuristic Biomechanical": "Futuristisch Biomechanisch",
+	"Futuristic Biomechanical Cyberpunk": "Futuristisch Biomechanischer Cyberpunk",
+	"Futuristic Cybernetic": "Futuristisch Kybernetisch",
+	"Futuristic Cybernetic Robot": "Futuristischer Kybernetischer Roboter",
+	"Futuristic Cyberpunk Cityscape": "Futuristische Cyberpunk-Stadtszene",
+	"Futuristic Futuristic": "Futuristisch",
+	"Futuristic Retro Cyberpunk": "Futuristischer Retro-Cyberpunk",
+	"Futuristic Retro Futurism": "Futuristischer Retro-Futurismus",
+	"Futuristic Sci Fi": "Futuristische Sci-Fi",
+	"Futuristic Vaporwave": "Futuristischer Vaporwave",
+	"Game Bubble Bobble": "Spiel Bubble Bobble",
+	"Game Cyberpunk Game": "Spiel Cyberpunk",
+	"Game Fighting Game": "Kampfspiel",
+	"Game Gta": "Spiel GTA",
+	"Game Mario": "Spiel Mario",
+	"Game Minecraft": "Spiel Minecraft",
+	"Game Pokemon": "Spiel Pokémon",
+	"Game Retro Arcade": "Retro-Arcade-Spiel",
+	"Game Retro Game": "Retro-Spiel",
+	"Game Rpg Fantasy Game": "RPG-Fantasy-Spiel",
+	"Game Strategy Game": "Strategiespiel",
+	"Game Streetfighter": "Spiel Streetfighter",
+	"Game Zelda": "Spiel Zelda",
+	"Misc Architectural": "Verschiedenes Architektonisch",
+	"Misc Disco": "Verschiedenes Disco",
+	"Misc Dreamscape": "Verschiedenes Traumlandschaft",
```

Comment:
Suggstion for "Dreamscape": Traumwelt

Reply:
I've not implemented this yet, as Traumwelt would be more akin to Dreamworld

Reply:
Possible also "Traumgestalt", "Traumbild". It is also not wrong to leave it as it is.

Reply:
I think I would leave it as it is for now, it can be changed later if it needs to be.